packages/react/src/utils/polymorphicComponent.test.tsx

```diff
@@ -143,3 +143,7 @@ const htmlAnchorRef = React.createRef<HTMLAnchorElement>();
     bananas
   </ConstrainedComponent>
 );
+
+test('polymorphicComponent', () => {
```

Comment:
I think instead of adding an empty test here, it would be better to exclude it from coverage.

```js
  collectCoverageFrom: [
    '<rootDir>/src/components/**/*.{tsx,ts}',
    '<rootDir>/src/utils/**/*.{tsx,ts}',
    '!<rootDir>/src/utils/**/polymorphicComponent.test.{tsx,ts}',
  ],
```